environment.yml

```diff
@@ -6,16 +6,14 @@ dependencies:
   - maven
   - make
   - cmake
-  - xtensor-zarr=0.0.4
+  - xtensor-zarr=0.0.7=*_1
```

Comment:
Can this also be `>=`? Otherwise, :tada:

Reply:
I changed it to `>=0.0.8|=0.0.7=*_1`.